runtime/queries/c/highlights.scm

```diff
@@ -117,9 +120,15 @@
  (preproc_arg)
  (preproc_defined)
 ]  @function.macro
-; TODO (preproc_arg)  @embedded
 
-(field_identifier) @property
+(((field_expression
+     (field_identifier) @property)) @_parent
+ (#not-has-parent? @_parent template_method function_declarator call_expression))
+
+(((field_identifier) @property)
+ (#has-ancestor? @property field_declaration)
+ (#not-has-ancestor? @property function_declarator))
```

Comment:
I think that the `has-ancestor`predicate is only implemented in nvim-treesitter and not in here, so that needs to be removed.

Reply:
Done :+1: